features/filmStrip/components/FilmStrip.js

```diff
@@ -40,6 +40,10 @@ class FilmStrip extends Component {
                                     participant={p}
                                     videoTrack={videoTrack}
                                     audioTrack={audioTrack}
+                                    audioMuted={!audioTrack ||
```

Comment:
Re-rendering wasn't being triggered without passing those values as props instead of making the check inside.